tests/components/deconz/test_binary_sensor.py

```diff
@@ -127,6 +130,7 @@ async def test_allow_clip_sensor(hass):
     assert "binary_sensor.clip_presence_sensor" in gateway.deconz_ids
     assert "binary_sensor.vibration_sensor" in gateway.deconz_ids
     assert len(hass.states.async_all()) == 4
+    assert len(gateway.entities[binary_sensor.DOMAIN]) == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)